src/containers/AccessPointDetails/components/General/index.js

```diff
@@ -324,12 +335,15 @@ const General = ({
           </Select>
         </Item>
 
+        {childProfiles.rfs.length > 0 && (
+          <Item label="RF Profile">{childProfiles.rfs.map(rf => rf.name)}</Item>
```

Comment:
you can just find the first of type rf in childProfiles instead, there's only going to be one